CHANGELOG.md

```diff
@@ -6,454 +6,494 @@ All notable changes to this project will be documented in this file.
-    - Performance reports for datasources
-    - Collect more metrics for datasource
-    - Deployment entity operations: Undeploy, redeploy
-    - Server operations: Reload, suspend, resume
-    - Live metrics for datasources and transactions
-    - Performance reports for middleware servers
-    - Support for alert profiles and alert automated expressions for middleware server
-    - Crosslink middleware servers with RHEV VMs
-    - Collect and display deployment status
-    - Datasources topology view
-    - Added missing fields in UI to improve user experience
-    - Middleware as top level menu item
-    - Default view for Middleware is datasource
-    - Change labels in middleware topology
-    - Added "Server State" into Middleware Server Details
-    - Enabled search for Middleware entities
-    - Users can add Middleware Datasources and JDBC Drivers
-    - Metrics for JMS Topics and Queues
-  - Kubernetes: Cross-linking with OpenStack instances
-  - Microsoft Cloud (Azure)
-    - Handle new events: Create Security Group, VM Capture
-    - Provider-specific logging
-    - Added memory and disk utilization metrics
-    - Support floating IPs during provisioning
-    - Load Balancer inventory collection for Azure
-    - Pagination support in armrest gem
-  - Microsoft Infrastructure (SCVMM): Set CPU sockets and cores per socket
-  - Networking
-    - Allow port ranges for Load Balancers
-    - Load Balancer user interface
-    - Separate Google Network Manager
-    - NFV: VNFD Templates and VNF Stacks
-    - Nuage: Inventory of Managed Cloud Subnets
-    -  Nuage policy groups added
+    - Added image tag names for Containers in vim performance state.
+    - Added a 'fixed_compute_metric' column to chargeback
+    - Added rate assigning by tags to container images
+    - Chargeback vm group by tag
+  - Notifications
+    - Dynamic substitution in notification messages
+    - Generate for Lifecycle events
+- Database Maintenance
+  - Hourly reindex: High Churn Tables
+  - Periodic full vacuum
+  - Configure in appliance console
+- Notification Backend
+  - Model for asynchronous notifications
+  - Authentication token generation for web sockets
+  - API for notification drawer
+- PostgreSQL High Availability
+  - Added [repmgr](http://repmgr.org/)  to support automatic failover
+    - Maintain list of active standby database servers
+    - Added [pg-dsn_parser](https://github.com/ManageIQ/pg-dsn_parser) for converting DSN to a hash
+  - DB Cluster - Primary, Standbys
+  - Uses [repmgr](http://www.repmgr.org/) (replication)
+  - Failover
+    - Primary to Standby
+    - Appliance connects to new primary DB
+    - Primary/Standby DB config in Appliance Console
+    - Database-only appliance config in Appliance Console
+    - Failover Monitor
+- Tenancy
+  - Post refresh hook on OpenStack provider
+  - Create provider base tenant under a root tenant
+  - Cloud Tenant tree generated under provider base tenant
+  - Create / Update / Delete
```

Comment:
I think these four bullets should be nested under OpenStack Tenant provider mapping.  By itself, this line makes it look like we added ` Create / Update / Delete` for tenancy which is wrong (I think) cc @gtanzillo